### PR TITLE
:book: Add media section to our docs

### LINKED ISSUE #601: :book: Add introduction videos and media section to our docs
**Is your feature request related to a problem? Please describe.**
We do have few videos and blog posts that can be helpful to our docs:

- https://www.youtube.com/watch?v=kiDQujibz2k
- https://www.youtube.com/watch?v=doXT1jxyZe0
- https://thenewstack.io/livin-kubernetes-on-the-immutable-edge-with-kairos-project/

Aside, also the slides used in the video should be public as well

**Describe the solution you'd like**
The introduction video sounds a great fit for our quickstart. I think we should also have a separate section with media resources, blog posts, howtos, etc.

### PR DESCRIPTION
**Pull request by mauromorales**
**What this PR does / why we need it**:

- Add media section
- update docsy
- fix issue with duplicated nave menu entry
- remove unexpected tag
- add blog section with first super short post about the new media section

**Which issue(s) this PR fixes** :
Fixes #601

### CODE REVIEW

**Comment by mudler** [comment]
@check-spelling-bot apply [updates](https://github.com/kairos-io/kairos/actions/runs/3833672132/attempts/1).

File: docs/content/en/docs/Media/_index.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+## Videos
+
+### Introduction to Kairos
+
+{{< youtube id="doXT1jxyZe0" title="Introduction to Kairos" >}}
+
+### Meet Kairos, an OSS project building the immutable Kubernetes edge
+
+{{< youtube id="kiDQujibz2k" title="Meet Kairos, an OSS project building the immutable Kubernetes edge" >}}
+Livin’ Kubernetes on the (Immutable) Edge with Kairos Project
```
Comment (mudler) [inline docs/content/en/docs/Media/_index.md:22]:
```suggestion
```

**Review by mudler — DISMISSED** [review]
Looks good here!

**Comment by mudler** [comment]
What I would add is also a link to this section in the quickstart and at the index of the docs so it doesn't get lost

File: docs/content/en/docs/Media/_index.md (hunk 1 above)
Comment (mauromorales) [reply docs/content/en/docs/Media/_index.md]: oops good catch

**Comment by mauromorales** [comment]
@mudler good idea, I've added them, should be good to go now

**Review by mudler — APPROVED** [review]
Looking good!

**Pull request merged**